src/qt/createwalletdialog.cpp

```diff
@@ -53,6 +53,13 @@ CreateWalletDialog::CreateWalletDialog(QWidget* parent) :
         }
     });
 
+    connect(ui->blank_wallet_checkbox, &QCheckBox::toggled, [this](bool checked) {
+        // Disable the disable_privkeys_checkbox when blank_wallet_checkbox is false
```

Comment:
Not sure if these kind of comments are helpful. You can see that stated fact by just looking at rather self-explanatory code, with descriptive variable names.

Maybe this message would be more informative:
```
//Having "Disable Private Keys" checked after unchecking "Make Blank Wallet" makes no sense. Disabling keys means wallet has to be blank.
```
Or similar.

Reply:
Actually, the comment is wrong as the `disable_privkeys_checkbox` is to be unchecked, not disabled :)

I think just removing it is fine.

Reply:
Removed the comment in 915e341